packages/web-components/src/mixins/uswds-input-width.scss

```diff
@@ -0,0 +1,25 @@
+:host .usa-input {
+  &--2xs {
```

Comment:
I had hoped that the width classes were included with the `usa-input` module but they have them in the forms module and nested under a `.usa-form` class. I did not think it made sense to wrap an individual component with `.usa-form` just so that we could access the width classes. Looking at the styles, it looks like this class should be used on a wrapping element around an entire form:

https://github.com/uswds/uswds/blob/develop/packages/usa-form/src/styles/_usa-form.scss 

I wonder if we should request that the USWDS team move the width classes into the  `usa-input` module instead? That way we wouldn't have to create a copy of the classes here.

Reply:
Or their own mixin that they can import in any module that needs them (usa-form & usa-input)? 🤷‍♀️  I'm not sure the answer to it but it's a bit inconvenient for us when we'd like to use them in an individual field rather than a full form. I'll see if I can create a ticket on their repo and strike up a convo about it.